src/plugins/home/server/services/sample_data/sample_data_installer.ts

```diff
@@ -173,17 +191,23 @@ export class SampleDataInstaller {
         await this.esClient.asCurrentUser.indices.createDataStream({
           name: index,
         });
+        if (aliases) {
+          await this.esClient.asCurrentUser.indices.updateAliases({
+            actions: Object.entries(aliases).map(([alias]) => ({
+              add: { index, alias },
+            })),
+          });
+        }
       } else {
         await this.esClient.asCurrentUser.indices.create({
           index,
-          body: {
-            settings: {
-              index: {
-                ...dataIndex.indexSettings,
-              },
+          settings: {
```

Comment:
The use of the 'body' key has been deprecated, so moving the nested keys to the top level object.

node_modules/@elastic/elasticsearch/lib/api/typesWithBodyKey.d.ts line 8755